.travis.yml

```diff
@@ -1,8 +1,5 @@
 language: node_js
 node_js:
-  - 0.10
-  - 0.12
-  - 4
-  - 6
-  - 8
-  - 9
```

Comment:
can we keep support down to Node.js 8 at least?

Reply:
Oh, I thought Node.js 8 was EOL?

Reply:
This was addressed in a recent PR, calling this resolved.